code-quality/accessibility/WCAG-testing.md

```diff
@@ -0,0 +1,102 @@
+# WCAG Testing Procedure
+
+This is the procedure that has been used to test the FUSD WordPress sites
+
+## Automated testing
+
+There are 3 primary tools used in testing, they are all Chrome browser extensions:
+
+- [WAVE Browser Extension for Chrome and Firefox](https://wave.webaim.org/extension/)
+- [SiteImprove Browser Extension](https://siteimprove.com/en-us/core-platform/integrations/browser-extensions/)
+- [Broken Link Checker](https://chrome.google.com/webstore/detail/broken-link-checker/nibppfobembgfmejpjaaeocbogeonhch)
+
+The first two go over the site currently loaded in the browser and point out possible issues. The third will test all links on a site. The first two tests should be done on as many pages as a user is likely to encounter upon a visit. If you can check all of them, great, but if your site has 100 pages, and 95 of those are the result of a template, you don't need to check all 100.
+
+The link checker, on the other hand, should be done on as much of the site as possible to avoid any broken links.
+
+### WAVE
+
+Make sure that you don't have any ancillary dashboards present on the site. For example, if you are logged into a WordPress admin account and then go to that page, you wil have the WP admin bar on the top of the screen. You don't want that; automated testers will hit that as well. Once that's taken care of, simply navigate to the website you wish to test and then click the WAVE icon in your browser:
+
+![WAVE Icon illustration](screens/wave-before.png)
+
+This will display several landmark on the screen and a summary on the left side
+
+![WAVE Active on a page](screens/wave-after.png)
+
+Clicking on the "Contrast" tab will display details on any contrast errors in your page. Clicking the flag icon on the left will display the details of errors/alerts on the page:
+
+![WAVE Details](screens/wave-details.png)
+
+This will display any Error and Alerts on the page. Errors should always be fixed. Alerts will can sometimes be ignored. For example, any link to a PDF file will prompt an alert because PDFs can't be tested automatically. Clicking the blue **i** next to the alert will give you a more detailed descripion as to why this is a problem and how it can be remedied.
+
+You don't need to worry about the other categories; the next automated test will present your ARIA landmarks in a way that's easier to evaluate, but this can be helpful to get an overview of the features and landmarks of your page.
+
+One neat feature is that when this is active, you will see on the bottom of the page a small green tab labeled **`<code>`**. Clicking this will pull up the pages code similar to the inspector. If you open this tab and then click on a particular error or alert on the left side, the code section will be highlighted like this:
```

Comment:
Let's change this to : `One neat feature that comes with WAVE displays a green tab labelled **`< code >`**...` So it flows better.

Also: `...will pull up the page's code similar...`